tool/generate.dart

```diff
@@ -13,36 +14,130 @@ void main() {
   // Remove comments and empty lines.
   data.removeWhere((row) => row.length < 3);
 
-  var file = new File("lib/src/generated.dart").openSync(mode: FileMode.write);
-  file.writeStringSync("""
-    // Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+  new Directory("lib/src/generated").createSync(recursive: true);
```

Comment:
At some point, it would be nice to get rid of these `new` keywords, but that should be a different patch.